src/api/java/com/minecolonies/api/colony/managers/interfaces/ICitizenManager.java

```diff
@@ -139,6 +139,12 @@ default ICitizenData spawnOrCreateCitizen(final ICitizenData data, @NotNull fina
      */
     int getMaxCitizens();
 
+    /**
+     * Get potential max citizens of the colony.
+     * @return the amount.
+     */
+    int getPotentialMaxCitizens();
```

Comment:
`The potential considers all available beds including not assigned guard towers.`
I’d add the same doc as the set method below is its more obvious